salt/_modules/metalk8s_checks.py

```diff
@@ -434,6 +442,31 @@ def route_exists(destination, raises=True):
     return error or True
 
 
+def containerd_filesystem(folder=None, raises=True, **kwargs):
+    error = None
+    folder = folder or "/var/lib/containerd"
+    while not os.path.exists(folder):
+        folder = os.path.dirname(folder)
+    res = __salt__["cmd.run_all"](f'df --output=fstype {folder}')
+    if res['retcode'] != 0:
+        error = f"Cannot check filesystem type of {folder}"
+    else:
+        # We use the exact output to prevent false positives
+        if "Type\nxfs" not in res['stdout']:
+            # Not an XFS filesystem, no need to check
+            return True
+        try:
+            infos = __salt__["xfs.info"](folder)
+            ftype = infos["naming"]["ftype"]
+            if ftype != "1":
+                error = f"XFS filesystem has ftype={ftype} expected 1"
```

Comment:
Can you add the "folder" in the error message
Something like that
```suggestion
                error = f"'Containerd XFS filesystem ({folder}) has ftype={ftype} expected 1"
```